sashimi-webapp/src/components/editor-viewer/Viewer.vue

```diff
@@ -19,37 +13,133 @@
 <script>
   import Vue from 'vue';
   import AsyncComputed from 'vue-async-computed';
+  import _ from 'lodash';
```

Comment:
That's its name

> Lodash draws most of its ideas from Underscore.js, and now receives maintenance from the original contributors to Underscore.js.
https://en.wikipedia.org/wiki/Lodash

`_` was an idea that came from the library called underscore.js.